src/components/button/Button.tsx

```diff
@@ -21,7 +21,9 @@ export interface ButtonProps {
   className?: string;
   title?: string;
   onClick?: (e: any) => void;
-
+  as?: any;
```

Comment:
I think it should be properly typed (give only button/anchor options since only those elements should be clickable)